### PR TITLE
Update ZeroTrustAnalyticsPlatform to 1.1.0

### PR DESCRIPTION
**Pull request by vimpadawan**
Simplify comment/log ingestion
* Remove audit log syncing. Information was confusing while investigating.
* Comments now show up as plaintext notes.
* Add original ZTAP user as comment footer.
* Deprecate ZTAPBuildTimeline. It was only used to differentiate comment/logs.
* Deprecate ZTAP Input Tag. It is only used in ZTAPBuildTimeline.

Minor improvements
* Add link back to original ZTAP alert.
* Change default mirroring direction to Both.
* Fix integration branding.
* Add field xsoar_trigger_kv to ease parsing.
* Clarify required synchronization fields for custom mappers in README.

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Address beta feedback.
Improve user experience for comments.
Minor improvements.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [x] Yes
       - NOTE this integration is in beta
       - Comments now ingest as text instead of json
       - Audit logs are no longer ingested from ZTAP
       - Build timeline is removed
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/README.md [hunk 1]
```diff
@@ -28,6 +28,18 @@ This integration was integrated and tested with version 2021-06-25 of ZeroTrustA
...
+### Custom Classifier
+If using a custom classifier the following fields are required for bidirectional sync
+
+| **Input Field** | **Output Field** |
+| --- | --- |
+| xsoar\_mirror\_id | dbotMirrorId |
+| xsoar\_mirror\_direction | dbotMirrorDirection |
+| xsoar\_mirror\_instance | dbotMirrorInstance |
+| xsoar\_mirror\_last\_sync | dbotMirrorLastSync |
+| xsoar\_mirror\_tags | dbotMirrorTags |
```
Comment (GuyAfik) [inline Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/README.md:41]: I think this should be done in the pack README

File: Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py [hunk 2]
```diff
@@ -365,85 +360,34 @@ def comments_to_notes(client: Client, comments: List):
...
+    def to_note(comment):
         occurred = comment["datetime_created"]
         oid = comment["id"]
-        readable = {
-            "occurred": occurred,
-            "contents": comment["comment"],
-            "type": "comment",
-            "files": filenames,
-        }
+        user_display = user_to_display(comment["user"])
```
Comment (GuyAfik) [inline Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py:366]:
```suggestion
        user_display = user_to_display(comment.get("user"))
```

File: Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py [hunk 3]
```diff
@@ -365,85 +360,34 @@ def comments_to_notes(client: Client, comments: List):
...
+        footer = f"\n\nSent by {user_display} via ZTAP"
         return {
             "Type": EntryType.NOTE,
             "ContentsFormat": EntryFormat.JSON,
             "Contents": comment,
-            "HumanReadable": readable,
-            "ReadableContentsFormat": EntryFormat.JSON,
+            "HumanReadable": comment["comment"] + footer,
```
Comment (GuyAfik) [inline Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py:372]:
```suggestion
            "HumanReadable": comment.get("comment") + footer,
```

You can also use f-strings here

File: Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py [hunk 4]
```diff
@@ -365,85 +360,34 @@ def comments_to_notes(client: Client, comments: List):
...
+            "ReadableContentsFormat": EntryFormat.TEXT,
```
Comment (GuyAfik) [inline Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py:373]: Why did you change it from a json to text?

File: Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py [hunk 5]
```diff
@@ -462,10 +406,11 @@ def get_notes_for_alert(
     comments = get_comments_for_alert(client, alert_id, last_update)
     entries.extend(comments_to_notes(client, comments))
 
-    audit_logs = get_audit_logs_for_alert(client, alert_id, last_update)
-    entries.extend(audit_logs_to_notes(client, audit_logs, last_update))
+    entries = sorted(entries, key=lambda x: x["sort"] if "sort" in x else "")
```
Comment (GuyAfik) [inline Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py:409]:
```suggestion
    entries = sorted(entries, key=lambda x: x.get("sort", ""))
```

File: Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py [hunk 6]
```diff
@@ -568,6 +513,26 @@ def was_alert_first_escalated(
...
+    instead of using field weights as it makes writing mapping for individual values easier.
+    """
+    trigger_event = None
+    for event in trigger_events:
+        if event.get("trigger"):
+            trigger_event = event
+            break
+    flattened = {}
+    if trigger_event:
+        for field in trigger_event["fields"]:
```
Comment (GuyAfik) [inline Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py:528]:
```suggestion
        for field in trigger_event.get("fields", []):
```

**Comment by GuyAfik** [comment]
Hi @vimpadawan, Thank you very much for your contributions and for the efforts that you made! See my comments. I think we can setup a short demo

File: Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py (hunk 4 above)
Comment (vimpadawan) [reply Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py:373]: Originally, we included both the audit logs and comments as notes, however the audit logs ended up cluttering incidents and made reading the actual comments more difficult. Moving the comments to plaintext allows reading the comment directly instead of having to expand the json object.

File: Packs/ZeroTrustAnalyticsPlatform/ReleaseNotes/1_1_0.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+- Add ZTAP Alert Link
+- Remove unused Escalate Organization
+
+#### Mappers
+##### ZeroTrustAnalyticsPlatform - Incoming Mapper
+- Add ZTAP Alert Link
+
+#### Playbooks
+##### ZTAP Alert
+- Add ZTAP Alert Link
```
Comment (tomer-pan) [inline Packs/ZeroTrustAnalyticsPlatform/ReleaseNotes/1_1_0.md:26]: @vimpadawan Couldn't find anywhere in the playbook anything related to the new alert link field. I did see that you removed one task which runs the `ZTAPBuildTimeline` script though.

Comment (vimpadawan) [reply Packs/ZeroTrustAnalyticsPlatform/ReleaseNotes/1_1_0.md]: `ZTAPBuildTimeline` was removed from the playbook, the alert was not added to the playbook. I updated the release notes.

**Review by GuyAfik — APPROVED** [review]
Nice work @vimpadawan! Thank you for your contribution!
